moj-fe/app/Http/Controllers/HealthController.php

```diff
@@ -3,17 +3,88 @@
 namespace App\Http\Controllers;
 
 use Symfony\Component\HttpFoundation\Response;
+use Illuminate\Support\Facades\Log;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Psr7;
+use GuzzleHttp\Client;
 
 class HealthController extends Controller
 {
-    /**
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
-    function checkHealth()
+
+    protected $client;
+
+    protected $response;
+
+    protected $cms;
+
+    protected $cmsHealth;
+
+    public function getResponse()
+    {
+        return $this->response;
+    }
+
+    public function getCmsHealth()
+    {
+        return $this->cmsHealth;
+    }
+
+    public function __construct()
+    {
+        $this->response = new Response();
+        $this->client = new Client(
+          [
+            'base_uri' => config('app.api_uri'),
+            'timeout' => config('app.timeout'),
```

Comment:
What's this one set to? might be worth overriding to 2 seconds regardless of what the global setting is

Reply:
I've hard coded the time to 30s.. it was at 60s